### ISSUE TITLE
Pretrained model link broken

### ISSUE THREAD

**Issue by ohayonguy** [KupynOrest/DeblurGAN#230]
Can someone please upload the pre-trained model to a new link?
Thanks!

**Comment by Ahmadreza1365** [comment]
I cannot upload the pretrained model here. I trained it yesterday on my own device. 

You can download the images from the link below and train it yourself. 

https://drive.google.com/file/d/1CPMBmRj-jBDO2ax4CxkBs9iczIFrs8VA/view

Please see: 

https://github.com/KupynOrest/DeblurGAN/issues/222#issuecomment-1704701870

**Comment by ohayonguy** [comment]
Hi, thanks! Would it be possible for you to send it via wetransfer.com ?
Or maybe upload it to the following link?
https://drive.google.com/drive/folders/1xQ4UJzChc4WY_9_HEL6rliJ1Ck7G7MNO?usp=sharing

This would really help a lot of people here I guess. If you already trained it, then this could be great :)
@Ahmadreza1365

**Comment by Ahmadreza1365** [comment]
@ohayonguy  
You're welcome.

Actually, the weights are already provided in the checkpoints folder, but I noticed that they do not work very well for the test images.

I don't have permission to upload them in the google drive you provided.
I uploaded the here.

https://drive.google.com/drive/folders/12vdMN72n0O8z9uMvPUMoTBwjUwdF9QLR?usp=sharing

you can replace it with the checkpoints folder you already have.

**Comment by ohayonguy** [comment]
@Ahmadreza1365 Okay thanks! Yes that's exactly what I encountered as well. The checkpoints provided in this github repo simply do not work well..
Thanks a lot for sharing the weights! Did you train the model according to the original implementation of the authors?

**Comment by Ahmadreza1365** [comment]
@ohayonguy I downloaded the images and divided the train and test images. 90% for train and 10% for test. Then Started the training. It took about 21 hours. My device info is: 

Intel(R) Core(TM) i9-10885H CPU @ 2.40GHz   2.40 GHz
32.0 GB (31.7 GB usable)
NVIDIA Quadro T2000 with Max-Q Design

**Comment by ohayonguy** [comment]
I am still getting quite awkward results, for instance on images from the GoPro data set. 

Here is the input blurry image:

![0_real_A](https://github.com/KupynOrest/DeblurGAN/assets/37740937/44dda3c3-4605-48d4-abb7-a69ecd3e9a12)

And the output of the model:
![0_fake_B](https://github.com/KupynOrest/DeblurGAN/assets/37740937/862271b8-6220-4e9d-b20e-489a6191e2b2)

Is this expected?
Thanks!

**Comment by Ahmadreza1365** [comment]
Yes, that is expected I believe. When the images are too blurry, it cannot handle them very well.

**Comment by Ahmadreza1365** [comment]
@ohayonguy 
I tried this one: 
https://github.com/VITA-Group/DeblurGANv2/tree/master
and the results are much better:
input: 
![12](https://github.com/KupynOrest/DeblurGAN/assets/89111238/6537b036-c715-4a17-88ca-ee49706038ec)

output:
![12](https://github.com/KupynOrest/DeblurGAN/assets/89111238/a959a456-0a79-4c8f-8569-9ad1a05771fe)

**Comment by ohayonguy** [comment]
@Ahmadreza1365 Indeed, I also tried it already.

**Comment by DungDT94** [comment]
> [@ohayonguy](https://github.com/ohayonguy) I tried this one: https://github.com/VITA-Group/DeblurGANv2/tree/master and the results are much better: input: ![12](https://github.com/KupynOrest/DeblurGAN/assets/89111238/6537b036-c715-4a17-88ca-ee49706038ec)
> 
> output: ![12](https://github.com/KupynOrest/DeblurGAN/assets/89111238/a959a456-0a79-4c8f-8569-9ad1a05771fe)

Hello, can you give me instructions to run this repo?
